src/slack_planning_bot/core/spec.clj

```diff
@@ -0,0 +1,65 @@
+(ns slack-planning-bot.core.spec
+  (require [slack-planning-bot.core.entity :as entity]
+           [clojure.spec.alpha :as s]))
+
+(s/def ::user-id string?)
+(s/def ::message string?)
```

Comment:
Message cannot be null, we dont allow sending null messages.

Reply:
According to [this](https://clojure.org/guides/spec) documentation, string? do not allow nil as a valid value